File: Linguaggi/Simulazioni/40/Python/Furgone.py

from Veicolo import Veicolo

class Furgone(Veicolo):
    
    def __init__(self, codice, targa, modello, marca, costo, categoria, posti, vano):
        super().__init__(codice, targa, modello, marca, costo)
        self.categoria = categoria
        self.posti = posti
        self.vano = vano
        
    def __str__(self):
        return "commerciale\t"+super().__str__() + "-\t-\t-\t"  +str(self.categoria)+"\t" +str(self.posti) +"\t"+str(self.vano)
        

File: Linguaggi/Simulazioni/40/Java/Furgone.java
public class Furgone extends Veicolo {
    protected String categoria;
    protected int posti;
    protected boolean vano;

    public Furgone(int codice, String targa, String modello, String marca, double costo, String categoria, int posti, boolean vano) {
        super(codice, targa, modello, marca, costo);
        this.categoria = categoria;
        this.posti = posti;
        this.vano = vano;
    }

    public String toString() {
        return "commerciale\t"
               +super.toString()+"\t-\t-\t"
               +this.categoria+"\t"
               +this.posti+"\t"
               +this.vano+"\t";
    }
}
